index.html

```diff
@@ -0,0 +1,268 @@
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <!-- ######################################### TOGETHER SECTION ######################################### -->
+    <div class="together_container">
+      <div class="together_background">
+        <img src="images/bubble-12.png" class="together_photo bubble12">
+        <img src="images/bubble-13.png" class="together_photo bubble13">
+        <img src="images/bubble-14.png" class="together_photo bubble14">
+        <img src="images/bubble-16.png" class="together_photo bubble16">
+        <img src="images/chat-group.png" class="together_photo chat">
+        <img src="images/bill-pay.png" class="together_photo bill">
+        <img src="images/laptop-poses.png" class="together_photo laptop">
+      </div>
+      <div class="together_title">
+        <h7>Let's Work Together</h7>
+        <h1>Need a successful project?</h1>
+      </div>
+      <div class="together_project">Estimate Project</div>
+    </div>
+    <!-- ######################################### FOOTER SECTION ######################################### -->
+    <div class="footer_container">
+      <div class="social_bar">
+        <img src="images/facebook.png">
+        <img src="images/twitter.png">
+        <img src="images/pinterest.png">
+        <img src="images/linkedin.png">
+        <img src="images/instagram.png">
+        <img src="images/yt.png">
+      </div>
+      <div class="main_footer_section">
+        <div class="left_section">
+          <div class="left_section_container">
+            <h1>Get in touch!</h1>
+            <h3>many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected</h3>
+          </div>
+          <div class="subscribe">
+            <div class="emial_inbox">
+              <img src="images/email.png" class="email">
+              Email address
+            </div>
+            <div class="subscribe_button">Subscribe</div>
+          </div>
+        </div>
+        <div class="right_section">
+          <div class="services">
+            <h4>Services</h4>
+            <h5>Logo & Branding</h5>
+            <h5>Website Development</h5>
+            <h5>Mobile App Development</h5>
+            <h5>Search Engine Optimalization</h5>
+            <h5>Pay-Per-Click</h5>
+            <h5>Social Media Marketing</h5>
+          </div>
+          <div class="resources">
+            <h4>Resources</h4>
+            <h5>Blog</h5>
+            <h5>Case Studies</h5>
+            <h5>Portfolio</h5>
+          </div>
+          <div class="support">
+            <h4>Support</h4>
+            <h5>Contact</h5>
+            <h5>Privacy Policy</h5>
+            <h5>Terms of Use</h5>
+          </div>
+        </div>
+      </div>
+      <div class="footer_end_section">
+        <div class="footer_end_section_left">
+          <img src="images/copyright.png" class="copyright">
+          All rights reserved by Me
```

Comment:
Luźny tekst pomiędzy znacznikami.

```suggestion
          <span>All rights reserved by Me</span>
```